rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -1310,11 +1340,14 @@ extern "C" rmw_ret_t rmw_destroy_node(rmw_node_t * node)
       common->pub, static_cast<void *>(&participant_msg), nullptr);
   }
 
-  rmw_free(const_cast<char *>(node->name));
-  rmw_free(const_cast<char *>(node->namespace_));
-  node->context->impl->fini();
-  rmw_node_free(node);
+  assert(node->context != nullptr);
```

Comment:
As with the other PRs around this, I'll suggest that we elide this check completely.  See the conversation at https://github.com/ros2/rmw_fastrtps/pull/408#discussion_r454341097 for more details on to why.

Reply:
Uh, I thought I had removed them all. There, see https://github.com/ros2/rmw_cyclonedds/pull/206/commits/49992cdf1ce22c9cf25f81262444eae0f6443362